apple/internal/local_provisioning_profiles.bzl

```
@@ -0,0 +1,161 @@
+"""# Rules for using locally installed provisioning profiles"""
+
+def _provisioning_profile_repository(repository_ctx):
+    system_profiles_path = "{}/Library/MobileDevice/Provisioning Profiles".format(repository_ctx.os.environ["HOME"])
+    repository_ctx.execute(["mkdir", "-p", system_profiles_path])
+    repository_ctx.symlink(system_profiles_path, "profiles")
+
+    repository_ctx.file("BUILD", """
```

This helps remove the empty line at the top of the generated BUILD file.
```suggestion
    repository_ctx.file("BUILD", """\
```